GithubJobApp/JobTableViewController.swift

```diff
@@ -19,138 +19,32 @@ class JobTableViewController: UITableViewController {
         return jobArray.count
     }
-
     
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
         guard let cell = tableView.dequeueReusableCellWithIdentifier("JobTableViewCell", forIndexPath: indexPath) as? JobTableViewCell else {
             return UITableViewCell()
         }
 
-        // Configure the cell...
-        let job = jobArray[indexPath.row]
-
-        cell.company.text = job.company
-        cell.title.text = job.title
-        let logoImageView = cell.companyLogo as! UIImageView
-        logoImageView.contentMode = UIViewContentMode.ScaleAspectFit
+        cell.job = jobArray[indexPath.row]
         
-        
-        if let actualCompanyLogoUrl = job.companyLogoUrl {
-            if self.companyLogoCache.objectForKey(actualCompanyLogoUrl) != nil, let logoAnyObject = companyLogoCache.objectForKey(actualCompanyLogoUrl), let logoImg = logoAnyObject as? UIImage {
-                print("using cached image")
-                logoImageView.image = logoImg
-            } else {
-                print("actualCompanyLogoUrl: " + actualCompanyLogoUrl + "\n")
-                
-                if let imgUrl = NSURL(string: actualCompanyLogoUrl) {
-                    let task = NSURLSession.sharedSession().dataTaskWithURL(imgUrl) {(data, response, error) in
-                        if error == nil {
-                            dispatch_async(dispatch_get_main_queue()) { () -> Void in
-                                if let actualData = data, logoImage = UIImage(data: actualData) {
-                                    logoImageView.image = logoImage
-                                    self.companyLogoCache.setObject(logoImage, forKey: actualCompanyLogoUrl as AnyObject)
-                                }
-                            }
-                        } else {
-                            if let actualError = error { print("error: \(actualError) \n") }
-                            dispatch_async(dispatch_get_main_queue()) { () -> Void in
-                                logoImageView.image = UIImage(named: "logo_placeholder")
-                                
-                            }
-                        }
-                    }
-                    task.resume()
-                }
-            }
-        }
         return cell
-        
     }
 
     override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
-        if(segue.identifier == "Detail View") {
-            if let detailViewController = segue.destinationViewController as? DetailViewController,
-                let row = self.tableView.indexPathForSelectedRow {
-                detailViewController.job = jobArray[row.row]
-            }
+        if let detailViewController = segue.destinationViewController as? DetailViewController,
+            let row = self.tableView.indexPathForSelectedRow {
```

Comment:
I'd call this `indexPath` instead of `row`.  The IndexPath object is one you'll see for both table views and collection views in iOS (and maybe else where).  For a table view, it contains both row and section information.  (Tables can have multiple sections, which in turn have multiple rows.)  Calling this an indexPath avoids the awkward `row.row` on the next line too.